Cargo.toml

```diff
@@ -44,12 +46,14 @@ html-escape = "0.2.13"
 near-sdk = { workspace = true, features = ["unit-testing"] }
 insta = { version = "1.31.0", features = ["json", "redactions"] }
 regex = "1"
-near-workspaces = { version = "0.14.0", features = ["unstable"], default-features = false  }
+near-workspaces = { version = "0.14.0", features = [
+    "unstable",
+], default-features = false }
 cargo-near-build.workspace = true
```

Comment:
this [dev-dependency] can be remove and near_workspaces re-exported version used instead